### ISSUE TITLE
Not able to build the Docker container

### ISSUE THREAD

**Issue by adgon** [urppeia/docker_bio_609_610_634#3]
This is what I did: 

-Downloaded and installed Docker for Windows
-Downloaded and installed Git Bash (it took me a while to figure out that I needed this as I had never used Git it was not obvious to me)
-I ran all this in Git Bash:

> adri@DESKTOP-IO03K8V MINGW64 ~
> $ cd Documents
> 
> adri@DESKTOP-IO03K8V MINGW64 ~/Documents
> $ git clone https://github.com/urppeia/docker_bio_609_610_634.git
> Cloning into 'docker_bio_609_610_634'...
> remote: Enumerating objects: 32, done.
> remote: Counting objects: 100% (32/32), done.
> remote: Compressing objects: 100% (27/27), done.
> remote: Total 32 (delta 12), reused 18 (delta 4), pack-reused 0
> Receiving objects: 100% (32/32), 6.85 KiB | 6.85 MiB/s, done.
> Resolving deltas: 100% (12/12), done.
> 
> adri@DESKTOP-IO03K8V MINGW64 ~/Documents
> $ cd docker_bio_609_610_634
> ./build.sh
> error during connect: This error may indicate that the docker daemon is not running.: Post "http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/build?buildargs=%7B%7D&cachefrom=%5B%5D&cgroupparent=&cpuperiod=0&cpuquota=0&cpusetcpus=&cpusetmems=&cpushares=0&dockerfile=Dockerfile&labels=%7B%7D&memory=0&memswap=0&networkmode=default&rm=1&shmsize=0&t=biodocker&target=&ulimits=null&version=1": open //./pipe/docker_engine: The system cannot find the file specified.
>

what should I do to keep Docker running?

**Comment by dktanwar** [comment]
Have you looked into this: https://stackoverflow.com/questions/40459280/docker-cannot-start-on-windows?

Please let me know if you are able to solve the issue with this solution, or I would try to find a Windows machine to test.

**Comment by adgon** [comment]
I tried several solutions proposed in that thread but they don't seem to work... The main problem is that Docker Desktop for some reason does not start.

**Comment by dktanwar** [comment]
OK. Well, then I try to find a Windows machine, and will keep you posted.

**Comment by adgon** [comment]
Finally, after uninstalling Docker, reset the computer (important to reset after uninstalling), installing Docker, reset agian. Docker seems to work. I managed to initiate the container building, but during step 11 I get this error


![error_execute_build_sh](https://user-images.githubusercontent.com/116676018/198127490-97bbd317-bf4b-4d29-95ba-563c0d2188bb.jpg)

**Comment by dktanwar** [comment]
Well, this is great. Could you please confirm version of Microsoft Windows you are using? 10 or 11?

**Comment by adgon** [comment]
I am using Windows 10

**Comment by dktanwar** [comment]
So, I made the installation easier. Please follow the instructions in the [README](https://github.com/urppeia/docker_bio_609_610_634/blob/master/README.md) now.
1. Open `Git Bash App`
2. Go to your directory: `cd /wherever/it/is/docker_bio_609_610_634`
3. `git pull`
4. `./win_login_student.sh`

Let me know.

**Comment by adgon** [comment]
Yes. I think that now it worked as I got the image and the container loaded in Docker.

Thank you!
